src/pages/Browse/index.tsx

```diff
@@ -40,7 +40,7 @@ type PropsFromState = {
   apiState: ApiState;
   file: VersionFile | null | void;
   fileIsLoading: boolean;
-  version: Version;
+  version: Version | void;
```

Comment:
I'm surprised that this can just be changed without any fallout in this component, but I guess it can!